src/Imath/ImathShear.h

```diff
@@ -332,14 +344,14 @@ template <class T>
 IMATH_HOSTDEVICE IMATH_CONSTEXPR14 inline T&
 Shear6<T>::operator[] (int i)
 {
-    return (&xy)[i]; // NOSONAR - suppress SonarCloud bug report.
+    return reinterpret_cast<T*> (this)[i]; // NOSONAR - suppress SonarCloud bug report.
```

Comment:
Remove the // NOSONAR comment. It shouldn't be necessary, and it didn't work anyway.

Reply:
done